src/docker-in-docker/.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Docker in Docker",
-	"image": "mcr.microsoft.com/devcontainers/base:buster",
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
```

Comment:
Can we bump the version in here for the changes to take effect? thanks
https://github.com/devcontainers/templates/blob/61ffd003da0e66d59a18db577ec9296594420bf5/src/docker-in-docker/devcontainer-template.json#L3

Reply:
Done!!